Tests/SwiftParserTest/ParserTests.swift

```diff
@@ -160,6 +160,8 @@ public class ParserTests: XCTestCase {
         || fileURL.absoluteString.contains("26773-swift-diagnosticengine-diagnose.swift")
         || fileURL.absoluteString.contains("parser-cutoff.swift")
         || fileURL.absoluteString.contains("26233-swift-iterabledeclcontext-getmembers.swift")
+        || fileURL.absoluteString.contains("26190-swift-iterabledeclcontext-getmembers.swift")
+        || fileURL.absoluteString.contains("26139-swift-abstractclosureexpr-setparams.swift")
```

Comment:
These aren't even *that* bad, pretty surprising to me that they cause a stack overflow. Isn't even 100 nested braces!

Reply:
The issue here appears to be debug builds. In release builds, these are just fine.